### PR TITLE
Remove downsampling code

### LINKED ISSUE #3958: Remove unused downsample bucket logic
The logic for downsampling of buckets was disabled a while ago since it caused more problems than it solved. The code is still there, though, which is why we should remove it.

### PR DESCRIPTION
**Pull request by philippotto**
This code was deactivated, but not removed, yet.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open some datasets and check general view capabilties

### Issues:
- fixes #3958 

------
- [X] Ready for review

### CODE REVIEW

File: frontend/javascripts/oxalis/model/bucket_data_handling/data_cube.js [hunk 1]
```diff
@@ -85,10 +85,8 @@ class DataCube {
     this.isSegmentation = isSegmentation;
 
     this.MAX_UNSAMPLED_ZOOM_STEP = resolutionsLength - 1;
+    this.ZOOM_STEP_COUNT = resolutionsLength;
```
Comment (daniel-wer) [inline frontend/javascripts/oxalis/model/bucket_data_handling/data_cube.js]: `MAX_ZOOM_STEP` and `MAX_UNSAMPLED_ZOOM_STEP` now are the same afaics. Do we even need the `MAX_UNSAMPLED_ZOOM_STEP` any longer?

**Comment by daniel-wer** [comment]
We could run the screenshot tests once the CI build was successful, just to be sure :)

`URL=... docker-compose up screenshot-tests`

File: frontend/javascripts/oxalis/model/bucket_data_handling/data_cube.js (hunk 1 above)
Comment (philippotto) [reply frontend/javascripts/oxalis/model/bucket_data_handling/data_cube.js]: Good catch! I had removed all usages, but forgot to remove the definition :see_no_evil:

**Comment by philippotto** [comment]
> We could run the screenshot tests once the CI build was successful, just to be sure :)
> 
> `URL=... docker-compose up screenshot-tests`

For the record: they succeeded :)

**Pull request merged**